test/test_util/boto3_util.py

```diff
@@ -20,7 +20,8 @@ def create_s3_resource(
         s3_resource = boto3.resource(
             "s3",
             aws_access_key_id=aws_credentials.access_key_id,
-            aws_secret_access_key=aws_credentials.secret_access_key
+            aws_secret_access_key=aws_credentials.secret_access_key,
+            region_name='us-east-1'
```

Comment:
Not that it is important but this region could be the same as the one defined in: emr_system